docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -1,111 +1,107 @@
     
-    ![](https://i.imgur.com/cGCSexB.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/fac1c6cb-b72f-47b1-a358-4ce41224a688)
+
+   Each session is limited to a few hours. 
+3. When you've mined enough Sepolia ETH (minimum of 0.05 Sepolia ETH), click on "Stop Mining" and claim your reward.
     
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/9ace2824-9030-4507-9b5f-50354bb99127)    
     
-Setting up a hosted node on Sepolia Testnet using Infura
----
+## 3. Access a node on the Sepolia testnet using Infura
 
-(Note: Infura provides a simple straight-forward way of setting up endpoints for interaction with the Ethereum chain and the Waku RLN smart contract without having to run a dedicated Ethereum node. Setting up infura is not mandatory. Operators concerned with the centralized aspect introduced by infura can setup their own node.)
+> _*NOTE:*_ Infura provides a simple way of setting up endpoints for interaction with the Ethereum chain and the Waku RLN smart contract without having to run a dedicated Ethereum node.
+> Setting up Infura is not mandatory. Operators concerned with the centralized aspect introduced by Infura should use their own node.
 
-1. Sign up to infura if you do not have an account already. [https://infura.io/register](https://infura.io/register)
+1. Sign up for Infura if you do not have an account already. [https://infura.io/register](https://infura.io/register)
     
     ![](https://i.imgur.com/SyLaG6s.jpg)
 
-2. After registering and verifying the account, create a new project using Ethereum and give it a name.
-    
-    ![](https://i.imgur.com/fJbNqwd.jpg)
+   Follow the instructions to register and verify the account.
+
+2. Click on "Create a new API key".
+
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/3cf1d537-9f73-4a6a-b1ef-bbac148bac47)
+
+4. Select "Web 3 API", give your API key a descriptive name and click on "Create".
+
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/f8c6c6b0-e55e-4681-8524-08303db11b9f)
 
-3. After creating the project, you will be presented with a dashboard like follows. Note that your Project Id and secret will be different.
+   You will be presented with a dashboard for the newly created key.
     
-   ![](imgs/infura-dashboard-mainnet.png)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/54c7c2da-f512-4c48-b5bb-ff86cae021c6)
 
-4. Select Sepolia network in Endpoints.
+4. Select "Sepolia" in the dropdown menu for Ethereum.
     
-    ![](https://i.imgur.com/RgmH7C1.png)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/945bf11e-15f6-48a9-9378-f7d39f73b02c)
 
-5. You can find the endpoints for the hosted node using https and wss. The wss endpoint is the relevant one for connecting the waku node to the RLN contract on Sepolia network. Like follows:
+   Both Https and WebSockets endpoints are available. Waku requires the WebSockets endpoint.
```

Comment:
I believe clicking `websocket` reset the network to `mainnet`. Might want to change the order